packages/zipkin/src/instrumentation/httpServer.js

```diff
@@ -54,13 +54,8 @@ class HttpServerInstrumentation {
           flags
         });
       });
-      return !this.tracer.supportsJoin
-        ? parentId.map(id =>
-            this.tracer.letId(id, () =>
-              this.tracer.createChildId()
-            )
-          )
-        : parentId;
+
+      return this.tracer.join(parentId);
```

Comment:
you'll notice all the below return a Some() and this is an internal function. Probably, unless I missed something, we can return just a trace ID here to simplify